spec/shared/controllers/shared_storage_manager_context.rb

```diff
@@ -2,17 +2,6 @@
   before do
     @ems_cloud = FactoryBot.create("ems_#{t}".to_sym,
                                     :name => "Test Cloud Manager")
-    if t == 'openstack'
-      @swift_manager    = @cinder_manager = nil
-      @storage_managers = @ems_cloud.storage_managers
-      @storage_managers.each do |sm|
-        if sm.type == "ManageIQ::Providers::Openstack::StorageManager::SwiftManager"
-          @swift_manager = sm
```

Comment:
I don't see `@swift_manager`, `@storage_managers` or `@cinder_manager` being used anywhere and cross-repo tests for ui-classic are green so I think these can all be simplified